frontend/src/components/DrawerInterior.tsx

```diff
@@ -23,12 +26,12 @@ import {
   FiberManualRecordRounded
 } from '@mui/icons-material';
 import FilterAltIcon from '@mui/icons-material/FilterAlt';
+import SaveIcon from '@mui/icons-material/Save';
```

Comment:
Combine the import of the newly added icons.
```suggestion
  FiberManualRecordRounded,
  FilterAlt as FilterAltIcon,
  Save as SaveIcon
} from '@mui/icons-material';
```

Reply:
This has been fixed.